components/tikv_util/src/sys/cgroup.rs

```diff
@@ -247,7 +248,12 @@ fn parse_memory_max(line: &str) -> i64 {
     if line == "max" {
         return -1;
     }
-    line.parse().unwrap()
+    match line.parse::<i64>() {
+        Ok(x) => x,
+        Err(e) if matches!(e.kind(), IntErrorKind::PosOverflow) => i64::MAX,
+        Err(e) if matches!(e.kind(), IntErrorKind::NegOverflow) => i64::MIN,
```

Comment:
What about treating any wrong value as no limit? `i64::MIN` is not so meaningful here, as any negative value is treated as no limit.

Reply:
I prefer to keep the raw value, because they will be log out at components/tikv_util/src/sys/mod.rs:72.

Reply:
If the value is negative, the logged value is also strange after casted to u64. Maybe it's better to fix the logging?

Reply:
The log line is also updated in the PR. It prints i64 instead of casted u64, so we can know the original value from that.